gcloud/logging/logger.py

```diff
@@ -41,3 +41,49 @@ def client(self):
     def project(self):
         """Project bound to the logger."""
         return self._client.project
+
+    @property
+    def full_name(self):
+        """Fully-qualified name used in logging APIs"""
+        return 'projects/%s/logs/%s' % (self.project, self.name)
+
+    def _require_client(self, client):
```

Comment:
`require` isn't quite the right word if we aren't going to error out in the absence?